vagrant/scripts/create_iml_diagnostics.sh

```diff
@@ -5,6 +5,6 @@ PREFIX=$1
 # Clean out any existing sos reports on this node
 yum install -y sos
 rm -f /var/tmp/*sosreport*
-sosreport --batch
+sosreport --batch -k docker.all=on -k docker.logs=on
```

Comment:
Isn't docker.all a superset of docker.logs?

Reply:
The docs indicate the following for docker.all:
```
docker.all                off             enable capture for all containers, even containers that have terminated
```

I ran a build yesterday with just `-k docker.all=on` and it included `inspect` information for all of the containers but it didn't include anything for the logs. I can provide a link if you want to see the output.

Reply:
Ok, fair enough.